db/migrate/20230719140256_add_updated_by_and_eligible_for_funding_updated_at_to_npq_application.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AddUpdatedByAndEligibleForFundingUpdatedAtToNPQApplication < ActiveRecord::Migration[7.0]
+  def change
+    add_column :npq_applications, :updated_by, :string
```

Comment:
Just because I can't see the ticket, do we intend to surface those values anywhere or is it explicitly for auditing purposes? If it's only for auditing purposes I wonder if we can use papertrail and add the `eligibility_for_funding` to the tracked fields?

If not and we intend to surface somewhere I suggest renaming to `eligible_for_funding_updated_by` as we're tracking the setting of that field only

Reply:
Yes, we do intend to surface these values. We want to show if an application was updated by an admin along with the timestamp. The answer to `where do we want to show it?` is currently not defined and we are working on grooming that part. 

I'll update the rest of the code according to the feedback.